packages/core-data/src/reducer.js

```diff
@@ -355,6 +358,11 @@ function entity( entityConfig ) {
 
 				return state;
 			},
+
+			// Add revisions to the state tree if the post type supports it.
+			...( entityConfig?.supports?.revisions
+				? { revisions: revisionsQueriedDataReducer }
+				: {} ),
```

Comment:
So just to get my understanding right. `revisions` are stored in the reducer separately from the regular entities right?

Reply:
Yes, at the same level as an entity type's queried data.

So under `entities.records.postType.page`:

```
{
    "queriedData": {  ...  },
    "edits": {},
    "saving": {},
    "deleting": {},
    "revisions": { ... }
}
```

And for global styles:

<img width="555" alt="Screenshot 2023-10-26 at 6 41 10 pm" src="https://github.com/WordPress/gutenberg/assets/6458278/e502ed8c-acbc-4d5d-bae1-9dfc5bf5c683">